### DISCUSSION TITLE
RangePicker option to have only one/current month in picker panel.

### DISCUSSION THREAD

**Discussion by yashsolanki22** [ant-design/ant-design#51204] [Q&A]
Hi folks, I would like to know if there is a for the RangePicker component to display only the current month's calender. By default it's showing two calenders, i.e. current month and next month.

**Comment by petercat-assistant** [comment]
Currently, the Ant Design `RangePicker` component is designed to show two calendars by default: the current month and the next month. Unfortunately, there isn't a built-in option to display only the current month's calendar in the `RangePicker`.

However, you can achieve this by customizing the component. One approach is to use the `Calendar` component instead of `RangePicker` and manage the date selection logic yourself. This way, you can control the display to show only the current month's calendar.

If you are looking for a more straightforward solution, you might consider submitting a feature request to the Ant Design team to add this functionality in future releases.

For further assistance, please describe your question in the comments and @petercat-assistant to start a conversation with me.

**Reply by yashsolanki22** [reply]
does using calender component provides support for selecting two dates to make it a range ??
